packages/backend/src/fixtures/responses/deletedObject.json

```diff
@@ -0,0 +1,6 @@
+{
+  "object": "domains",
+  "id": "123123123",
+  "slug": "",
+  "deleted": true
+}
```

Comment:
:wrench: This is not a generic "DeletedObject". It's the JSON payload for a "domain" deleted object. If we want to make this re-usable we should leave any mention of "domain" out of the payload.

How about we remove this file and instead move the fake response inside the test? See the other comment above.